aircon/aircon.py

```diff
@@ -157,10 +157,19 @@ def queue_command(self, name: str, value) -> None:
     # Device mode is set using t_control_value
     if issubclass(data_type, enum.Enum):
       data_value = data_type[value]
-    elif data_type is int and type(value) is str and '.' in value:
-      # Round rather than fail if the input is a float.
-      # This is commonly the case for temperatures converted by HA from Celsius.
-      data_value = round(float(value))
+    elif data_type is int:
```

Comment:
No reason to check for "." here, if it's an int value that doesn't have a decimal place the float logic won't affect it.